src/frontend/src/components/NavBar/components/RightNav/components/UserMenu/index.tsx

```diff
@@ -20,6 +20,14 @@ const UserMenu = ({ user }: UserMenuProps): JSX.Element => {
     setAnchorEl(null);
   };
 
+  const handleCookieSettings = () => {
+    // OneTrust _should_ always be loaded by time user doing real interaction
```

Comment:
Where does OneTrust load from?  I assume it's not blocked by ad blockers?

Reply:
OneTrust loads from a script hosted by OneTrust's CDN. Honestly, it probably is blocked by certain ad blockers, but I think we would consider that an alternative from of opt-out. I don't think OT would normally get blocked, but very aggressive ones might.

Reply:
I realize now that if it has been blocked, that "Cookie Settings" menu option will permanently do nothing for that user. I had just thought of it as a wacky edge case where if the OneTrust script hadn't loaded yet or failed to load due to a network error, it would be broken temporarily, but probably fixed after waiting a couple more seconds or on a refresh. I should bring this up to Olivia and/or Christine that if OneTrust is fully blocked, the Cookie Settings button will do nothing. Maybe we should have an `alert` message pop up to cover this case? I expect it's very rare, but thanks for making me think about it.